sklearn/svm/_classes.py

```diff
@@ -1316,3 +1319,227 @@ def probA_(self):
     @property
     def probB_(self):
         return self._probB
+
+
+class SVDD(OutlierMixin, BaseLibSVM):
+    """Support Vector Data Description for Unsupervised Outlier Detection.
+
+    Estimate the support of a high-dimensional distribution by finding the
+    tightest soft hypersphere around a data set, which permits at most a
+    fraction ``nu`` (``0 < nu <= 1``) of the data as outliers.
+
+    The implementation is based on libsvm.
+
+    Read more in the :ref:`User Guide <svm_outlier_detection>`.
+
+    Parameters
+    ----------
+    kernel : {'linear', 'poly', 'rbf', 'sigmoid', 'precomputed'}, default='rbf'
+         Specifies the kernel type to be used in the algorithm.
+         It must be one of 'linear', 'poly', 'rbf', 'sigmoid', 'precomputed' or
+         a callable.
+         If none is given, 'rbf' will be used. If a callable is given it is
+         used to precompute the kernel matrix.
+
+    degree : int, default=3
+        Degree of the polynomial kernel function ('poly').
+        Ignored by all other kernels.
+
+    gamma : {'scale', 'auto'} or float, default='scale'
+        Kernel coefficient for 'rbf', 'poly' and 'sigmoid'.
+
+        - if ``gamma='scale'`` (default) is passed then it uses
+          1 / (n_features * X.var()) as value of gamma,
+        - if 'auto', uses 1 / n_features.
+
+        .. versionchanged:: 0.22
+           The default value of ``gamma`` changed from 'auto' to 'scale'.
```

Comment:
Please remove this `.. versionchanged` statement as the full class was not present prior to v0.22.